Classes/Domain/Field.php

```diff
@@ -66,7 +66,7 @@ class Field extends AbstractFormObject
     public function __construct(Form $form = null, string $name = null, $targetValue = null, $multiple = false)
     {
         $this->form = $form;
-        $this->name = $name;
+        $this->name = $name ?? '';
```

Comment:
Guess the name property can then even be typehinted as `string` only. But that's just a nitpick.

Reply:
I am reluctant since this would change the method signature. Probably not relevant but changing signatures has a history of beeing unexpectedly breaky.